src/consumer/frame.js

```diff
@@ -141,6 +149,26 @@ class Frame extends EventEmitter {
     if (this.wrapper.parentNode === this.container) {
       this.container.removeChild(this.wrapper);
```

Comment:
After removing the wrapper do we want to set it to null?

Reply:
the wrapper is set to null in clean up method.